app/assets/javascripts/live-site-search.js

```diff
@@ -13,157 +13,159 @@
     $resultCount: false,
     $ariaLiveResultCount: false,
 
-    init: function(){
-      if(GOVUK.support.history()){
-        liveSearch.$form = $('.js-live-search-form');
-        liveSearch.$ariaLiveResultCount = $('.js-aria-live-count');
-        if(liveSearch.$form){
-          liveSearch.$resultsBlock = $('.js-live-search-results-block');
+    init: function () {
+      if (GOVUK.support.history()) {
+        liveSearch.$form = $('.js-live-search-form')
+        liveSearch.$ariaLiveResultCount = $('.js-aria-live-count')
+        if (liveSearch.$form) {
+          liveSearch.$resultsBlock = $('.js-live-search-results-block')
 
-          liveSearch.action = liveSearch.$form.attr('action') + '.json';
+          liveSearch.action = liveSearch.$form.attr('action') + '.json'
 
-          liveSearch.saveState();
+          liveSearch.saveState()
 
-          liveSearch.$form.on('change', 'input[type=checkbox]', liveSearch.checkboxChange);
-          $(window).on('popstate', liveSearch.popState);
+          liveSearch.$form.on('change', 'input[type=checkbox]', liveSearch.checkboxChange)
+          $(window).on('popstate', liveSearch.popState)
         }
       } else {
-        $('.js-live-search-fallback').show();
+        $('.js-live-search-fallback').show()
       }
     },
-    popState: function(event){
-      if(event.originalEvent.state){
-        liveSearch.saveState(state);
-        liveSearch.updateResults();
-        liveSearch.restoreCheckboxes();
-        liveSearch.pageTrack();
+    popState: function (event) {
+      if (event.originalEvent.state) {
+        liveSearch.saveState(event.originalEvent.state)
```

Comment:
Can you explain this change please?

Reply:
`state` was being flagged as a global, and I figured if we're explicitly checking that `event.originalEvent.state` exists, that's the value we'd want to be using...?

Passing `state` alone probably worked by coincidence, but better to be explicit. A bit like writing `if (window.alert) { alert() }`.